src/bot/index.ts

```diff
@@ -58,11 +58,11 @@ export async function bot({ currency, family, mutation }: Arg = {}) {
   const maybeCurrency = currency
     ? findCryptoCurrencyByKeyword(currency)
     : undefined;
-  const maybeFilterOnlyFamily = family;
+  const maybeFilterOnlyFamily = family ? family.split(",") : undefined;
 
-  for (const family in allSpecs) {
-    const familySpecs = allSpecs[family];
-    if (maybeFilterOnlyFamily && maybeFilterOnlyFamily !== family) {
+  for (const fam in allSpecs) {
```

Comment:
More `spec` than `fam` ideally ^^

Reply:
it's family because allSpecs is something like etc.... 
```
{ algorand, bitcoin, ethereum, cosmos }
```

But that shouldn't matter a lot because fam is just use in these two lines (and family was use before as an arg)